frontend/packages/kubevirt-plugin/src/components/modals/cdrom-vm-modal/cdrom-modal.scss

```diff
@@ -0,0 +1,24 @@
+.kubevirt-cdrom-modal__restart-alert {
+  padding-bottom: var(--pf-global--spacer--sm);
+}
+
+.kubevirt-cdrom-modal__row-name {
+  font-weight: var(--pf-global--FontWeight--bold);
+  margin-bottom: var(--pf-global--spacer--sm);
+}
+
+.kubevirt-cdrom-modal__row {
+  display: grid;
```

Comment:
I'm still not sure why we need custom classes for these. We could use `pf-l-grid` with `pf-m-gutter`. Maybe we can talk further about this if that's helpful.